components/Layout/Navigation.js

```diff
@@ -19,6 +19,15 @@ const CurrentUser = (props) => {
   }
 }
 
+const getFilter = () => {
+  if(window.location.pathname ==='/')
+    return null;
+  else if(window.location.pathname === '/options')
+    return <Link className={s.filter} to="/options">Search</Link>
+  else
+    return <Link className={s.filter} to="/options">Filter</Link>
+}
```

Comment:
I'm not crazy about two different cases going to the same page, just having different text. Should search just be within the current results set instead of going to the options page?